modulesets/gstreamer.modules

```diff
@@ -78,6 +80,13 @@
     <branch repo="gstreamer" module="gst-devtools.git" checkoutdir="gst-devtools" subdir="validate"/>
   </autotools>
 
+  <autotools id="gst-player" autogenargs="--disable-gtk-doc">
+    <dependencies>
+      <dep package="gstreamer"/>
```

Comment:
I think gst-player requires gst-plugins-base at least, doesn't it? If so, please add it. How about gst-plugins-good?